media-video/ffmpeg/ffmpeg7-7.1.recipe

```diff
@@ -0,0 +1,358 @@
+		ccArgs="--cc=$cc --cxx=g++${secondaryArchSuffix/_/-}
+			--ld=gcc${secondaryArchSuffix/_/-}
+			--host-ld=gcc${secondaryArchSuffix/_/-}"
+
+		# Hack up base headers to make them think we're on GCC2 ABI
+		mkdir -p include_hacks include_hacks/os include_hacks/config
+		pushd include_hacks
+			rm -f os/BeBuild.h config/HaikuConfig.h
+			cp /system/develop/headers/os/BeBuild.h os
+			cp /system/develop/headers/config/HaikuConfig.h config
+			sed -i 's/__GNUC__ == 2/1/g' os/BeBuild.h config/HaikuConfig.h
+		popd
+
+		# Compile gcc_runtime.c (subset of libgcc that GCC 13 utilizes here)
+		$cc -fvisibility=hidden -c $portDir/additional-files/gcc_runtime.c \
+			-o gcc_runtime.o
+
+		baseLDFLAGS="-B/system/bin/${gccSuffix/_/}"
+		ccArgs="$ccArgs --host-ldflags=\"${baseLDFLAGS}\""
+		export CFLAGS="-isystem$(pwd) -isystem$(pwd)/os -isystem$(pwd)/config"
+		export LDFLAGS="$baseLDFLAGS $(pwd)/gcc_runtime.o"
+		extraArgs="--disable-asm"
+	else
+		extraArgs="--enable-libass \
+			--enable-libdav1d \
+			--enable-libfdk-aac \
+			--enable-fontconfig \
+			--enable-libfreetype \
+			--enable-libfribidi \
+			--enable-libgme \
+			--enable-gmp \
+			--enable-libopenjpeg \
+			--enable-libopenmpt \
+			--enable-libsnappy \
+			--enable-libsoxr \
+			--enable-libvpx \
+			--enable-libwebp \
+			--enable-libxml2 \
+			--enable-lzma \
+			--enable-pic"
+	fi
+
+	# not an autotools configure
+	./configure \
+		$ccArgs \
+		--prefix=$prefix \
+		--bindir=$binDir \
+		--datadir=$dataDir/$portName \
+		--incdir=$includeDir \
+		--libdir=$libDir \
+		--shlibdir=$libDir \
+		--mandir=$manDir \
+		--disable-debug \
+		--disable-stripping \
+		--disable-static \
+		--enable-shared \
+		--enable-version3 \
+		--enable-libmp3lame \
+		--enable-libopus \
+		--enable-openssl \
+		--enable-libspeex \
+		--enable-libtheora \
+		--enable-libvorbis \
+		$extraArgs
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	prepareInstalledDevelLibs $ffmpegLibs
+	fixPkgconfig
+
+	# The pkgconfig file reference other libraries using the wrong paths, which
+	# creates a lot of confusion. Fix them so correct paths are used.
+	local develPackageName="${portName}_devel-$portFullVersion"
+	local packageLinksDir=$(dirname $portPackageLinksDir)
+	local linksDir="$packageLinksDir/${develPackageName}/devel~libspeex$secondaryArchSuffix/$relativeDevelopLibDir"
+	sed -i -e "s,^\(Libs.private.*\)-L.*speex.* \(-lspeex.*\),\1-L$linksDir \2," \
+		$developLibDir/pkgconfig/libavcodec.pc \
+		$developLibDir/pkgconfig/libavdevice.pc \
+		$developLibDir/pkgconfig/libavfilter.pc \
+		$developLibDir/pkgconfig/libavformat.pc
+
+	# Move html docs to the right place.
+	mkdir -p $docDir
+	mv $prefix/share/doc/ffmpeg $docDir
+	rm -r $prefix/share
+
+	# Split out libs and executables which need libSDL
+	packageEntries tools $binDir $manDir/man1
+	packageEntries avdevice $libDir/libavdevice*
+
+	# devel package
+	packageEntries devel \
+		$dataDir/ffmpeg7$secondaryArchSuffix/examples \
+		$manDir/man3 \
```

Comment:
With $manDir/man1 moved to the tools package, you can drop $manDir/man3 and $docDir here and replace with $documentationDir, solves empty references in the base package (probably for ffmeg6 too).